.rubocop_todo.yml

```diff
@@ -1,11 +1,16 @@
 # This configuration was generated by
 # `rubocop --auto-gen-config`
-# on 2021-09-14 12:45:14 UTC using RuboCop version 1.19.1.
+# on 2021-09-28 04:57:36 UTC using RuboCop version 1.21.0.
 # The point is for the user to remove these configuration records
 # one by one as the offenses are removed from the code base.
 # Note that changes in the inspected code, or installation of new
 # versions of RuboCop, may require this file to be generated again.
 
+# Offense count: 1
+Lint/UselessAssignment:
```

Comment:
Usually this cop right? Are you sure there is no useless assignment?

Reply:
Added comment instead of useless assignment
https://github.com/ONLYOFFICE/onlyoffice_api_gem/pull/335/commits/bc26f939ad9d3baba06dc0f733e404a80c9bc7df